src/utils/networkUtils.js

```diff
@@ -31,7 +31,14 @@ export const INFURA_JSON_RPC_URLS = {
 	42: `https://kovan.infura.io/v3/${INFURA_PROJECT_ID}`,
 };
 
-export const SUPPORTED_WALLETS = ['Metamask', 'Trezor', 'Ledger', 'Coinbase', 'WalletConnect'];
+export const SUPPORTED_WALLETS_MAP = {
+	METAMASK: 'Metamask',
+	TREZOR: 'Trezor',
+	LEDGER: 'Ledger',
+	COINBASE: 'Coinbase',
+	WALLET_CONNECT: 'WalletConnect',
```

Comment:
In next PRs we can start dropping the hard coded string in favour of `SUPPORTED_WALLETS_MAP`